### DISCUSSION TITLE
how to hide replicate

### DISCUSSION THREAD

**Discussion by nirav-arsenaltech** [laravel/nova-issues#6014] [Q&A] [answered]
- Laravel Version: 8.83.27
- Nova Version: 4
- PHP Version: 8.2.4
- Operating System and Version: windows, ubuntu
- Browser type and version: chrome

### Description:

how to hide 3 dots ?(i don't know what it is, maybe replicate)
![Screenshot Capture - 2023-11-01 - 16-23-24](https://github.com/laravel/nova-issues/assets/147684940/a9f527e9-c2d6-4b93-ba24-e2394f5865ea)

**Comment by crynobone** [comment, marked as answer]
Please refer to the documentation: https://nova.laravel.com/docs/resources/authorization.html#policies

**Reply by nirav-arsenaltech** [reply]
i can't find the solution

**Comment by mariusghitoiu** [comment]
The replicate action is indeed removed when replicate method returns false in resource policy 
```
    public function replicate($request)
    {
        return false;
    }
```
but the ... remain, without any action. 


https://github.com/laravel/nova-issues/assets/147089/4d380e24-68c6-421b-be93-baa15cb7b08a

**Reply by vesper8** [reply]
This didn't work for me.

What worked was this:

```

    public function authorizedToReplicate(Request $request)
    {
        return false;
    }
```
